pkg/daemon/ceph/util/util.go

```diff
@@ -28,10 +28,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+// DefaultServiceAccount is a  service-account used for components that do not specify a dedicated service-account.
 const (
-	RBDSysBusPathDefault = "/sys/bus/rbd"
-	RBDDevicesDir        = "devices"
-	RBDDevicePathPrefix  = "/dev/rbd"
+	RBDSysBusPathDefault  = "/sys/bus/rbd"
+	RBDDevicesDir         = "devices"
+	RBDDevicePathPrefix   = "/dev/rbd"
+	DefaultServiceAccount = "rook-ceph-default"
```

Comment:
nit: let's put this in an operator package somewhere, perhaps in `pkg/operator/ceph/controller/spec.go`.

Reply:
> nit: let's put this in an operator package somewhere, perhaps in `pkg/operator/ceph/controller/spec.go`.

Okay, sure.